v2/common/src/main/java/com/google/cloud/teleport/v2/cdc/merge/DataStreamBigQueryMerger.java

```diff
@@ -51,9 +52,11 @@
 
   public DataStreamBigQueryMerger(
       GcpOptions options,
+      String projectId,
```

Comment:
Similar to the other constructor, could you please make sure there's only one parameter per line for consistency?